cypress/e2e/homepage.cy.ts

```diff
@@ -0,0 +1,61 @@
+describe("homepage spec", () => {
+  it("renders correctly", () => {
```

Comment:
Please add jest command to cypress and use in data-testid instead

Reply:
Fixed